openlibrary/plugins/upstream/checkins.py

```diff
@@ -183,13 +199,13 @@ def POST(self):
         if i.is_update:
             if goal < 0:
                 raise web.badrequest(
-                    'Reading goal update must be 0 or a positive integer'
+                    message='Reading goal update must be 0 or a positive integer'
                 )
         elif not goal or goal < 1:
-            raise web.badrequest('Reading goal must be a positive integer')
+            raise web.badrequest(message='Reading goal must be a positive integer')
 
         if i.is_update and not i.year:
-            raise web.badrequest('Year required to update reading goals')
+            raise web.badrequest(message='Year required to update reading goals')
 
         user = get_current_user()
```

Comment:
Do we need to check that user exists before trying to split it?

Reply:
Yes, good catch!  I'll update this branch quickly....

Reply:
Just fixed it.  The `authorized_for` decorator also handles this currently.